src/platform/platform_sdl_input.cpp

```diff
@@ -0,0 +1,115 @@
+#include "platform_sdl.h"
+#include "platform_input.h"
+#include "utils/log.h"
+#include <SDL.h>
+#include <SDL_gamecontroller.h>
+
+namespace platform
+{
+
+namespace sdl {
+
+void
+handleEvent(const SDL_Event *event)
+{
+   switch (event->type) {
+   case SDL_QUIT:
+      shouldQuit = true;
+      break;
+   }
+}
+
+} // namespace sdl
+
+namespace input
+{
+
+enum {
+   MAX_SDL_JOYSTICKS = 16  // Arbitrary limit.
+};
+
+static const ControllerHandle keyboardHandle = 1;
+
+static const ControllerHandle joystickHandleStart = 10;
+
+static const ControllerHandle joystickHandleEnd = 10 + MAX_SDL_JOYSTICKS;
+
+struct JoystickData
+{
+   SDL_Joystick *handle;
+};
+static JoystickData joystickData[MAX_SDL_JOYSTICKS];
+
+static const uint8_t *keyboardState;
+static int keyboardStateLen;
+
+
+ControllerHandle
+getControllerHandle(const std::string &name)
+{
+   if (name.compare("keyboar") == 0) {
```

Comment:
Fixed, thanks for catching that. I may have broken it while adding joystick code and not noticed since I now have decaf configured for joystick input.